kernel/src/process.rs

```diff
@@ -1986,4 +1966,64 @@ impl<C: 'static + Chip> Process<'a, C> {
         let current_state = self.state.get();
         current_state != State::StoppedFaulted && current_state != State::Fault
     }
+
+    /// Update the maximum stack depth that the process has ever used.
+    fn debug_update_max_stack_depth(&self) {
+        if config::CONFIG.debug_processes == false {
+            return;
+        }
+        self.debug_update_max_stack_depth_using_pointer(self.current_stack_pointer.get());
+    }
```

Comment:
Does this compile differently than:

```rust
    fn debug_update_max_stack_depth(&self) {
        if config::CONFIG.debug_processes {
             self.debug_update_max_stack_depth_using_pointer(self.current_stack_pointer.get());
        }
    }
```

Generally, I'd say writing it this way reads clearer.

Reply:
Done.